### ISSUE TITLE
Can't find jasmine

### ISSUE THREAD

**Issue by ocoka** [gruntjs/grunt-contrib-jasmine#282]
Running "jasmine:main" (jasmine) task
>> Jasmine version: undefined does not exist in npm!

**Comment by ocoka** [comment]
No jasmine downloaded, it just exits silently and says _SpecRunner.html not found

**Comment by steveoh** [comment]
I have noticed this as well. I believe it is a race condition. [buildSpecRunner](https://github.com/gruntjs/grunt-contrib-jasmine/blob/master/tasks/jasmine.js#L101) is an async function but is not awaited. It is async because the [pacote](https://github.com/gruntjs/grunt-contrib-jasmine/blob/master/tasks/lib/jasmine.js#L52) package downloads jasmine.

So I think puppeteer starts before jasmine is downloaded. Once that happens, a folder is created which makes the code think jasmine is there, it's actually an empty folder, and you get your issue.

I've tried awaiting the buildspecrunner function but that causes other problems. So, I need a little help debugging this.
